### PR TITLE
Add Zepto plugin files & missing versions

### PR DESCRIPTION
**Pull request by hare1039**
Pull request for issue: #10612
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/madrobby/zepto
 * Official website (optional, not the repository): http://zeptojs.com
 * NPM package url (optional): https://www.npmjs.com/package/zepto
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 590
   - Count of stars: 12300
   - Count of forks: 3329
 * NPM download stats (optional):
   - Downloads in the last day: 73
   - Downloads in the last week: 1284
   - Downloads in the last month: 5468

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [ ] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by hare1039** [comment]
@cdnjs/intern3 Please help me on reviewing this PR. Thanks

Note I minify all the files using `--no-map` because original files doesn't have `.map` files

**Comment by hare1039** [comment]
Ping @sufuf3 @extend1994 @clairetsai818

**Comment by hare1039** [comment]
@sufuf3 I think it is not necessary to switch to git auto-update as latest version are both `1.2.0`.
All previous versions are included, I don't see a reason to use git auto-update.

File: ajax/libs/zepto/package.json [hunk 1]
```diff
@@ -20,6 +20,12 @@
       "files": [
         "**/*"
       ]
+    },
+    {
+      "basePath": "src",
+      "files": [
+        "!(zepto).js"
```
Comment (pvnr0082t) [inline ajax/libs/zepto/package.json]: @hare1039 If we don't need `zepto.js` or `zepto.min.js`, `"!(zepto)*"` would be more flexible.

**Comment by hare1039** [comment]
@pvnr0082t @extend1994 I add the `v` in the commit message and removed the trailing `.js` in package.json. Please help me review this PR again. Thanks!

**Comment by pvnr0082t** [comment]
@hare1039 Please rebase with the latest master branch. Thanks.

**Comment by hare1039** [comment]
@pvnr0082t I rebased this PR.

**Comment by hare1039** [comment]
ping

**Comment by PeterDaveHello** [comment]
@hare1039 be polite.

**Comment by sufuf3** [comment]
@hare1039 Would you fix this PR as this comment https://github.com/cdnjs/cdnjs/pull/10992#pullrequestreview-34916656?
Thank you.

**Comment by sufuf3** [comment]
ping @hare1039

**Comment by sashberd** [comment]
@PeterDaveHello @sufuf3 @extend1994 I cannot rebase this PR to latest master. Please help me with it or just close it.

**Comment by sufuf3** [comment]
Help ping @hare1039

**Comment by sashberd** [comment]
@sufuf3 Help me to rebase this PR and I will fix it.

**Comment by sufuf3** [comment]
@sashberd There you go.
Thanks for your help.

**Comment by sufuf3** [comment]
@sashberd You're right.
Also, IMO, maybe you could open another PR to handle this, because there is no response from contributor since 27 Apr 2017.
What do you think?

cc @PeterDaveHello, Not sure this is a good idea. But maybe this is good for Alex's contribution.

Many thanks.

**Comment by PeterDaveHello** [comment]
@sufuf3 sure, you are a experienced maintainer now 👍

**Comment by sashberd** [comment]
@sufuf3 The PR was updated. I cannot change the reviewers on the top(bug of browser I think). FYR

File: ajax/libs/zepto/package.json [hunk 2]
```diff
@@ -20,6 +20,12 @@
       "files": [
         "**/*"
       ]
+    },
+    {
+      "basePath": "src",
```
Comment (extend1994) [inline ajax/libs/zepto/package.json]: How about using `""`?

File: ajax/libs/zepto/package.json [hunk 3]
```diff
@@ -20,6 +20,12 @@
...
+      "files": [
+        "!(zepto)*"
```
Comment (extend1994) [inline ajax/libs/zepto/package.json]: Then zepto.js can be reserved. Suppose this zepto.js is different from the one under `dist`.

File: ajax/libs/zepto/package.json [hunk 4]
```diff
@@ -20,6 +20,12 @@
...
+      ]
     }
   ]
-}
\ No newline at end of file
+}
```
Comment (extend1994) [inline ajax/libs/zepto/package.json]: Mistake?

File: ajax/libs/zepto/package.json [hunk 5]
```diff
@@ -22,4 +22,4 @@
       ]
     }
   ]
-}
+}
```
Comment (extend1994) [inline ajax/libs/zepto/package.json]: Mistake in https://github.com/cdnjs/cdnjs/pull/10992/commits/7f80fba6fb45c6ecd06b631ca38c97e2e13ad736?

**Comment by sashberd** [comment]
@extend1994 I did not understand toy review exactly, so:
1) basePath of src -> ""
2) files of src -> zepto.js
3) and problem with space

Am I right?

**Comment by extend1994** [comment]
@sashberd 
```js
{
  "basePath": "src",
  "files": [
    "!(zepto)*"
  ]
}
```
to
```js
{
  "basePath": "",
  "files": [
    "src/*"
  ]
}
```

in order to host `dist/zepto.js` & `src/zepto.js`

point 3 & 4, yes

File: ajax/libs/zepto/package.json (hunk 4 above)
Comment (sashberd) [reply ajax/libs/zepto/package.json]: I just ran fixtool and this space is added by it... Have no idea what it was added. I push PR now and if the problem still exists in your opinion please help to fix. Thanks

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

File: ajax/libs/zepto/package.json (hunk 4 above)
Comment (extend1994) [reply ajax/libs/zepto/package.json]: @sashberd Let's try to undo this change! A good chance to do practice.

**Comment by sashberd** [comment]
@extend1994 The PR was updated. FYR

File: ajax/libs/zepto/package.json (hunk 4 above)
Comment (sashberd) [reply ajax/libs/zepto/package.json]: @extend1994 I removed this new line, but again it was added by fixtool of cdnjs tools. FYI

**Comment by sashberd** [comment]
@extend1994 Now the CI build is failed after I fixed what you asked in your review, please help to fix if you do not mind
Thanks

File: ajax/libs/zepto/package.json [hunk 6]
```diff
@@ -20,6 +20,12 @@
...
+    },
+    {
+      "basePath": "",
+      "files": [
+        "src/*"
+      ]
     }
   ]
-}
+}
```
Comment (sufuf3) [inline ajax/libs/zepto/package.json]: @sashberd Would you please try to use `./tools/fixFormat.js` under cdnjs folder to fix this problem?
Many thanks.

Comment (sashberd) [reply ajax/libs/zepto/package.json]: I tried, nothing happened!

Comment (extend1994) [reply ajax/libs/zepto/package.json]: @sashberd I tried `fixFormat.js` but the situation didn't happen, please try again, maybe you can just don't use the script.

**Comment by sashberd** [comment]
@extend1994 @sufuf3 The PR was updated. I ran fixtool on both commits in this PR. If there are any other problems with package.json fell free to fix them.

**Comment by sashberd** [comment]
@xtuc @MattIPv4 What would you like to do with this one?

**Comment by MattIPv4** [comment]
Per our other conversations, we are moving away from manually committing any version files ourselves and should only ever be commiting updates to package.json files, so that the bot can then handle getting version files for libraries.

Can that be done here, just updating package.json?

**Comment by sashberd** [comment]
Moved to #13695

**Pull request closed without merging**